openerp/addons/base/rng/view.rng

```diff
@@ -251,6 +251,7 @@
             <rng:optional><rng:attribute name="string"/></rng:optional>
             <rng:optional><rng:attribute name="colors"/></rng:optional>
             <rng:optional><rng:attribute name="fonts"/></rng:optional>
+            <rng:optional><rng:attribute name="limit"/></rng:optional>
```

Comment:
This allows passing a limit of "onion" or "1340 Shield" which we probably don't want to allow.